src/components/wallet/ActivityDetailModal.js

```diff
@@ -16,6 +16,7 @@ const StyledContainer = styled.div`
     h2 {
         text-align: center;
         margin-top: 20px;
+        color: #24272a !important;
```

Comment:
!important 😱  Is this a smell that we've got styles that are too broad in their scope?